specification/redisenterprise/resource-manager/Microsoft.Cache/preview/2021-02-01-preview/redisenterprise.json

```diff
@@ -1122,6 +1125,65 @@
           "nextLinkName": null
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Cache/redisEnterprise/{clusterName}/databases/{databaseName}/forceUnlink": {
+      "post": {
+        "tags": [
+          "RedisEnterprise",
+          "Databases"
+        ],
+        "operationId": "Databases_ForceUnlink",
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "azure-async-operation"
+        },
+        "x-ms-examples": {
+          "How to unlink a database during a regional outage": {
+            "$ref": "./examples/RedisEnterpriseDatabasesForceUnlink.json"
+          }
+        },
+        "description": "Forcibly removes the link to the specified database resource.",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ClusterNameParameter"
+          },
+          {
+            "$ref": "#/parameters/DatabaseNameParameter"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ForceUnlinkParameters"
+            },
+            "description": "Information identifying the database to be unlinked."
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Force unlink operation succeeded. The unlinked database must be deleted before it can be recreated and linked again."
+          },
+          "202": {
+            "description": "Force unlink operation successfully enqueued; follow the Location header to poll for final outcome."
```

Comment:
nit: Update description to 'follow the Azure-AsyncOperation header to poll for final outcome.'
In x-ms-long-running-operation-options above line no. 1138, as you specified final state is via azure-async-operation, the final response will be available at the uri pointed to by the header Azure-AsyncOperation.
Refer doc: http://azure.github.io/autorest/extensions/#x-ms-long-running-operation-options